src/materials/MeshPhongMaterial.js

```diff
@@ -177,6 +177,23 @@ THREE.MeshPhongMaterial.prototype.toJSON = function () {
 	if ( this.blending !== THREE.NormalBlending ) data.blending = this.blending;
 	if ( this.side !== THREE.FrontSide ) data.side = this.side;
 
+	if(this.map !== null && this.map !== THREE.Texture ) data.map = this.map.uuid;
```

Comment:
No I think @mrdoob is right there: the ```this.map !== THREE.Texture``` part does not make sense AFAIK.

I tested in console and ran in the debugger to be sure .. I don't think comparing to THREE.Texture like that is ever true, so the ```!==``` is always true. The code works because it's just a null check I think, I mean only the first part does anything: ```this.map !== null```

So if such a check is needed it needs to be done otherwise. Or if it's unnecessary and current behaviour is fine, AFAIK the 'dead code' would be good to remove for clarity. Which one is the case, I don't know :o

This is how I checked in console:

```
> mesh.material.map
THREE.Texture {uuid: "DB8EF18D-1797-4CD7-A26A-949271BDCBDE", name: "", sourceFile: "../../examples/textures/terrain/grasslight-thin.jpg", image: img, mipmaps: Array[0]…}

> mesh.material.map instanceof THREE.Texture
true

> mesh.material.map !== THREE.Texture
true

> mesh.material.map !== null
true

> mesh.material.map === THREE.Texture
false

> mesh.material.map == THREE.Texture
false
```

And in the debugger saw how the uuid was assingned when the map was there.